recipes/libta-lib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libta-lib" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-src.tar.gz  # [unix]
+    sha256: 9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651                    # [unix]
+    folder: unix                                                                                # [unix]
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-msvc.zip
+    sha256: 2dae7a1391222f0a7e601ecda654f93cebedfdb37afeb30df2721a3b0a717875
+    folder: win
```

Comment:
The LICENSE file is only found in the windows zip, so I'm downloading it in all cases to find that.